test/e2e/scheduler_predicates.go

```diff
@@ -31,12 +31,24 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+// Returns a number of currently running and not running Pods.
+func getPodsNumbers(pods *api.PodList) (runningPods, deadPods int) {
```

Comment:
deadPods is not the best name in my opinion (i.e. those are also pending pods). Maybe just "notRunningPods" ?